### ISSUE TITLE
Usability issue: Default Storage Location appears twice

### ISSUE THREAD

**Issue by joel-simpson** [artefactual/archivematica#964]
In the Store AIP microservice, users may be given the choice of which location to store their AIP. The same choice is also available in the Admin / Processing configuration screen. 
![screen shot 2018-03-08 at 5 48 44 pm](https://user-images.githubusercontent.com/17187309/37182075-aaf6a108-22fd-11e8-8528-90abba572d5f.png)

All of the AIP storage locations are defined in the Storage Service. One of those locations must be marked as the "Default". 

The UI shows all of the locations, but it also provides the option "Default". Showing the "Default" location as an option to be selected is confusing, because that location is already listed by whatever name it has been called in the 

In a standard installation, only one AIP storage location is created, with the description "Store AIP in standard Archivematica Directory". This is also marked as the default. The screenshot above is the result - two options in the UI that actually refer to only one directory. 

A simpler solution would be for the UI to show one location but with the word "default" shown after it. e.g. "Standard Archivematica Directory (default)". 
Users wouldn't have to remember which directory is marked as the default, and they wouldn't be confused into thinking that the "Default" option was actually a unique location.

**Comment by sromkey** [comment]
I agree with Joel's suggestion, or some other indication about which location is earmarked as Default would make this feature less confusing.
